src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -1,8 +1,13 @@
 import { Shipping_order } from "__generated__/Shipping_order.graphql"
 import React, { Component } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
+import { Join } from "Styleguide/Elements/Join"
```

Comment:
@zephraph - what does `Join` do?

Reply:
It inserts a separator between all of its direct children. In FP circles this might be called 'interpose'. I suppose in JS land the closest thing is Array.join, which does optionally insert a separator, but also concatenates the result into a string at the end.

Reply:
Ahh, makes sense 👍

Reply:
Yep. It wasn't one of my more creative naming days.